sdk/nodejs/provider/provider.ts

```diff
@@ -147,6 +147,11 @@ export interface Provider {
      */
     version: string;
 
+    /**
+     * The JSON-encoded schema for this provider's package.
```

Comment:
This would be a great place to link to the meta-schema (the schema of the schema), if we have it.